packages/node-resolve/README.md

```diff
@@ -47,40 +47,44 @@ Then call `rollup` either via the [CLI](https://www.rollupjs.org/guide/en/#comma
 ### `mainFields`
 
 Type: `Array[...String]`<br>
-Default: `['module', 'main']`
+Default: `['module', 'main']`<br>
+Possible values: Array of any values from `['browser', 'jsnext', 'module', 'main']` in any order.
 
-The fields to scan in a package.json to determine the entry point if this list contains "browser", overrides specified in "pkg.browser" will be used
+The fields to scan in a package.json to determine the entry point. Order of fields is significant, first found field would become the resolved entry point. If this list contains "browser", overrides specified in `pkg.browser` will be used.
```

Comment:
```suggestion
Specifies the properties to scan within a `package.json`, used to determine the bundle entry point. The order of property names is significant, as the first-found property is used as the resolved entry point. If the array contains `'browser'`, key/values specified in the `package.json` `browser` property will be used.
```

Reply:
I didn't backtick package.json, because it wasn't backticked previously. I use backticks to specify field names, like `pkg.browser` (such pkg names was already used before changes).

Reply:
Hm I think it deserves to be highlighted in that way. I'd like to see this change before I can approve.